tests/ballerina-integration-test/src/test/resources/filter/filterservices/01_multiple_filters_attribute_share_test.bal

```diff
@@ -22,7 +22,8 @@ import ballerina/log;
 FilterDto dto = { authenticated: true, username: "abcde" };
 
 public type Filter1 object {
-    public function filterRequest(http:Listener listener, http:Request request, http:FilterContext context) returns boolean {
+    public function filterRequest(http:Caller caller, http:Request request, http:FilterContext context)
```

Comment:
Shall we align the returns for 4 spaces from the function name here and all other places?

```suggestion
    public function filterRequest(http:Caller caller, http:Request request, http:FilterContext context)
                        returns boolean
```